source/_docs/z-wave/controllers.markdown

```diff
@@ -11,18 +11,13 @@ This Z-Wave integration is deprecated and replaced with a [new implementation ba
 
 ## Supported Z-Wave USB Sticks & Hardware Modules
 
-You need to have a compatible Z-Wave stick or module installed. This needs to be a *static controller*, which most Z-Wave sticks and modules will be. If yours is a *bridge* device then it won't work with [OpenZWave](http://openzwave.com/), which is what provides Home Assistant's Z-Wave capabilities. USB sticks using the new 700 series Z-Wave platform are not compatible. The following devices have been confirmed to work:
+You need to have a compatible Z-Wave stick or module installed. This needs to be a *static controller*, which most Z-Wave sticks and modules will be. If yours is a *bridge* device then it won't work with [OpenZWave](http://openzwave.com/), which is what provides Home Assistant's Z-Wave capabilities. The following devices have been confirmed to work:
 
-<div class='note'>
-  
-There are [known compatibility issues](https://www.raspberrypi.org/forums/viewtopic.php?f=28&t=245031#p1502030) with older hardware versions of the Aeotec stick not working on the Raspberry Pi 4. Aeotec has released a 2020 hardware revision ZW090-A/B/C Gen5+ with Pi 4 compatibility. Both hardware revisions are still being sold, make informed purchasing decisions if using paired with a Pi 4.
-
-</div>
-
-- Aeotec Z-Stick Series 5
+- Aeotec Z-Stick Series 5 (see note below)
```

Comment:
This is not the Z-Wave JS documentation, but the legacy Z-Wave integration documentation, which is based on OpenZWave 1.4, not Z-Wave JS.

Reply:
Sorry about that. I thought this page was for both integrations since it is linked in the documentation for Z-Wave JS (requirement 2 here https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/zwave_js.markdown#requirements )

Is there a separate list for the Z-Wave JS supported controllers anywhere that we can update that link to?

Reply:
I have now been searching a while for a controller support list for Z-Wave JS without success. Maybe that is why the OpenZWave list is linked.

If I get some advice / pointers where such a list would be suitable to create I could start with it. Would that fit best on a separate page or inline in the current zwave_js.markdown file?

Reply:
Oh darn, you are absolutely right!

Let stick with this for now 👍